### ISSUE TITLE
Localization files not found

### ISSUE THREAD

**Issue by jluttine** [martinrotter/rssguard#175]
#### Brief description of the issue.

Building 3.5.5 fails.

#### How to reproduce the bug?

1. Compile rssguard 3.5.5.

#### What is the expected result?

Successful build.

#### What actually happened?

Build failed.

```
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_cs.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_da.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_de.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_en.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_en_GB.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_es.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_fr.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_he.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_id.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_it.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_ja.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_lt.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_nl.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_pl.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_pt.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_sv.qm'
RCC: Error in 'resources/rssguard.qrc': Cannot find file '../localization/rssguard_zh.qm'
```

#### Other information (logs, see Wiki)

Unfortunately, it's a bit difficult to show the full commands that I used to compile. I'm packaging this application for NixOS. If needed, I can try to get more information.

**Comment by martinrotter** [comment]
See [project file](https://github.com/martinrotter/rssguard/blob/master/rssguard.pro#L17). Let me know if it helped.

**Comment by jluttine** [comment]
Ok, thanks. Just making sure that it is supposed to be such that in the repo the suffices are `.ts` but `rssguard.qrc` searches for files with suffix `.qm`? So the build process should convert .ts to .qm?

**Comment by martinrotter** [comment]
Exactly, "lrelease" converts ts -> qm and places those files in expected folders. Let me know if it works now, I could maybe fix the process so that "lrelease" step would not be needed.

**Comment by martinrotter** [comment]
OK, it should be now fixed in a better way. `lrelease` step now should not be needed.

https://github.com/martinrotter/rssguard/blob/master/rssguard.pro#L10

https://github.com/martinrotter/rssguard/commit/3fdb3b10bfd575d413302ce9c66e0785325701d2

**Comment by jluttine** [comment]
Ok, thanks! And yes, adding `lrelease rssguard.pro` fixed the issue for building 3.5.5.
